src/app/containers/CpsAsset/index.jsx

```diff
@@ -40,58 +38,72 @@ const SingleContentWrapper = styled.div`
   }
 `;
 
-const FeaturesAnalysis = ({ content }) => {
+const CpsAsset = ({
+  title,
+  content,
+  enableGridWrapper,
+  a11yAttributes,
+  listTransform,
+  singleTransform,
+}) => {
   const { script, service, dir } = useContext(ServiceContext);
```

Comment:
if we have access to the `script, service, dir` props from where this component is being used would it be worth passing them as props so this component doesn't also depend on the `ServiceContext` as well

Reply:
One advantage of using contexts is to reduce the props in components to a minimum. Does adding the props explicitly offer much of an advantage?

Reply:
my concern was how they (parent and child components) are both tightly coupled to the same context. If we were testing both in isolation we would also have to mock that context in the tests.